3.Tests/UI/StepDefinitions/User/LoginStepDefinition.cs

```diff
@@ -0,0 +1,68 @@
+﻿using TechTalk.SpecFlow;
+using Todoly.Core.Helpers;
+using Todoly.Core.UIElements.Drivers;
+using Todoly.Core.UIElements.Interfaces;
+using Todoly.Core.UIElements.Web;
+using Todoly.Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "User Login")]
+[TestFixture]
+public class LoginChromeTests
+{
+    private readonly LoginPage _loginPage;
+    private HomePage? _homePage;
+    private readonly ScenarioContext _scenarioContext;
+
+    public LoginChromeTests(ScenarioContext scenarioContext)
+    {
+        _loginPage = new LoginPage();
+        _scenarioContext = scenarioContext;
+    }
+
+    [Given(@"the user navigates to the URL")]
+    public void GivenTheUserNavigatesToTheURL()
+    {
+        GenericWebDriver.Instance.Navigate().GoToUrl(ConfigModel.HostUrl);
+    }
+
+    [When(@"the user clicks on Login")]
+    public void WhenTheUserClicksOnLogin()
+    {
+        _loginPage.LoginButton.Click();
+    }
+
+    [When(@"introduces his credentials")]
+    public void WhenIntroducesHisCredentials()
+    {
+        _loginPage.EmailTextField.Clear();
+        _loginPage.EmailTextField.Type(_loginPage.EmailCredentials);
```

Comment:
Clearing an input field could be done in the implementation of the Type method (since we want to follow best practices, we will always clear before sending keys)
```csharp
public void Type(string keys)
{
    try
    {
        WebElement.Clear();
        WebElement.SendKeys(keys);
        GenericWebDriver.Wait.Until(ExpectedConditions.TextToBePresentInElementValue(WebElement, keys));
    }
    catch (WebDriverTimeoutException error)
    {
        System.Console.WriteLine($"Keys not sent.");
        throw error;
    }
}
```